### PR TITLE
Use random UUIDs for asset keys (instead of content hashes)

### PR DESCRIPTION
**Pull request by AurelienGasser**
Companion PRs:

- https://github.com/SubstraFoundation/substra/pull/235
- https://github.com/SubstraFoundation/substra-backend/pull/342
- https://github.com/SubstraFoundation/substra-chaincode/pull/128

**Please see the main PR in substra-backend for a detailed description of the changes:** https://github.com/SubstraFoundation/substra-backend/pull/342

---

The mechanism adding random variations to assets specs is now obsolete (because unnecessary) and has been removed.

### CODE REVIEW

**Comment by AurelienGasser** [comment]
Note that the CI tests won't pass until the substra PR is merged https://github.com/SubstraFoundation/substra-tests/pull/153

**Comment by AurelienGasser** [comment]
@inalgnu @jmorel @samlesu bump

**Review by samlesu — APPROVED** [review]
LGTM

**Review by inalgnu — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -106,3 +106,5 @@ venv.bak/
 # IDE
 .idea
 .vscode
+
+local-worker/
```
Comment (jmorel) [inline .gitignore:110]: Thank you :)

**Review by jmorel — APPROVED** [review]

**Pull request merged**